packages/cloud-cognitive/src/components/CreateModal/CreateModal.test.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
Rather than `fireEvent.click`, our guidelines suggest using `userEvent` from `@testing-library/user-event`. This has been updated in `ExampleComponent.test.js` and is the recommended way to go now.

```js
import userEvent from '@testing-library/user-event';
```